site/homepage/content/sandbox.md

```diff
@@ -39,9 +39,12 @@ All services are exposed via the same ports as used in the guide.
   Telemetry and/or Event APIs, make sure to not set a trust store explicitly but instead set the *tlsEnabled* property to `true`.
   The command line client binary is available from the [downloads page]({{< relref "downloads#binaries" >}}) and
   can be used to consume telemetry/event messages from the sandbox as below:
-  
-        java -jar hono-cli-*-exec.jar --hono.client.host=hono.eclipse.org --hono.client.port=15671 --hono.client.tlsEnabled=true --hono.client.username=consumer@HONO --hono.client.password=verysecret --spring.profiles.active=receiver`
-  Note that  only the *receiver* profile is activated but not the *ssl* profile.
+
+  ~~~sh
+  # in directory where the hono-cli-*-exec.jar file has been downloaded to
+  java -jar hono-cli-*-exec.jar --hono.client.host=hono.eclipseprojects.io --hono.client.port=15671 --hono.client.tlsEnabled=true --hono.client.username=consumer@HONO --hono.client.password=verysecret --spring.profiles.active=receiver
+  ~~~
+  Note that only the *receiver* profile is activated but not the *ssl* profile.
```

Comment:
this doesn't seem related to your PR, is it?

Reply:
That's right. Was caused by a merger mishap. Removed